src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpWinForms.cs

```diff
@@ -114,6 +116,7 @@ public void RenameControl()
             VisualStudio.Editor.EditWinFormButtonEvent(buttonName: "SomeButton", eventName: "Click", eventHandlerName: "SomeButtonHandler");
             // Rename the control
             VisualStudio.Editor.EditWinFormButtonProperty(buttonName: "SomeButton", propertyName: "Name", propertyValue: "SomeNewButton");
+            Thread.Sleep(TimeSpan.FromSeconds(20));
```

Comment:
❗ This should be removed from the PR

```suggestion
```

Reply:
Woops, left-over from testing.  Will remove